components/b2b/Lists/CreateList/CreateList.tsx

```diff
@@ -0,0 +1,235 @@
+        productCode: product?.productCode as string,
+      },
+      quantity: 1,
+    }
+    items.push(item)
+
+    // converting product to CrWishlistItem
+    const crWishlistProduct: CrWishlistItem = {
+      quantity: 1,
+      product: {
+        productCode: product?.productCode,
+        price: product?.price as CrProductPrice,
+        imageUrl:
+          (product?.content?.productImages?.length as number) > 0
+            ? (product?.content?.productImages?.[0]?.imageUrl as string)
+            : '',
+        name: product?.content?.productName as string,
+        description: product?.content?.productFullDescription as string,
+      },
+    }
+
+    setListState((currentState) => ({ ...currentState, items: items }))
+    // setting state to show the products below
+    setProductList((currentVal) => [...currentVal, crWishlistProduct])
+  }
+
+  const handleListNameChange = (e: string, userEnteredValue: string) => {
+    setListState((currentVal) => ({ ...currentVal, name: userEnteredValue }))
+  }
+
+  const handleDeleteItem = (id: string) => {
+    const items: any = listState.items.filter((item) => {
+      return item.product.productCode !== id
+    })
+    setListState((currentState) => ({ ...currentState, items: items }))
+    setProductList((currentState) =>
+      currentState.filter((item: CrWishlistItem) => item.product?.productCode !== id)
+    )
+  }
+
+  const handleChangeQuantity = (id: string, quantity: number) => {
+    const item = listState.items.find((item) => item.product.productCode === id)
+    if (item?.quantity) item.quantity = quantity
+  }
+
+  return (
+    <>
+      <Box sx={{ width: '100%' }}>
+        {mdScreen && (
+          <Button
+            data-testid="my-account-button"
+            sx={{ paddingLeft: 0, fontSize: '14px', color: '#000' }}
+            onClick={() => {
+              router.push('/my-account')
+            }}
+            startIcon={<ArrowBackIosIcon sx={{ width: '14px' }} />}
+          >
+            {t('my-account')}
+          </Button>
+        )}
+        <Typography
+          variant="h3"
+          sx={{ ...styles.heading, margin: mdScreen ? '20px 0' : '0px 10px 0px 0px' }}
+          fontWeight={'bold'}
+        >
+          {mdScreen ? (
+            <>
+              <Box sx={{ fontSize: '28px', marginRight: 'auto', display: 'inline' }}>
+                {t('create-new-list')}
+              </Box>
+              <Box sx={{ display: 'inline' }}>
+                <Button
+                  variant="contained"
+                  color="secondary"
+                  type="button"
+                  onClick={() => {
+                    onCreateFormToggle(false)
+                  }}
+                >
+                  {t('cancel')}
+                </Button>
+                <Button
+                  variant="contained"
+                  type="submit"
+                  form="wishlist-form"
+                  sx={{ boxShadow: 'none', marginLeft: '9px' }}
```

Comment:
use gap on parent Box, remove marginLeft. Remove the boxShadow style as well.

Reply:
done